src/sentry/integrations/slack/message_builder/notifications/issues.py

```diff
@@ -31,3 +31,26 @@ def build(self) -> SlackBody:
             notification=self.notification,
             recipient=self.recipient,
         ).build()
+
+
+class ActiveReleaseIssueNotificationMessageBuilder(SlackNotificationsMessageBuilder):
+    def __init__(
+        self,
+        notification: ProjectNotification,
+        context: Mapping[str, Any],
+        recipient: Team | User,
+    ) -> None:
+        super().__init__(notification, context, recipient)
+        self.notification: ProjectNotification = notification
+
+    def build(self) -> SlackBody:
+        group = getattr(self.notification, "group", None)
+        return SlackIssuesMessageBuilder(
+            group=group,
+            event=getattr(self.notification, "event", None),
+            tags=self.context.get("tags", None),
+            rules=getattr(self.notification, "rules", None),
+            issue_details=True,
+            notification=self.notification,
+            recipient=self.recipient,
+        ).build()
```

Comment:
This seems to be where we build up the slack message. Do we need to add anything here to complete the mocks?

Reply:
not at the moment i don't think